etc/org.opencastproject.ui.metadata.CatalogUIAdapterFactory-episode-common.cfg

```diff
@@ -85,7 +85,7 @@ property.description.readOnly=false
 property.description.required=false
 property.description.order=2
 
-# A Duration property
+# A Duration property (Display is optional)
```

Comment:
@karendolan Note that the setting "required" is not just about indicating "required" vs "optional" in the UI - both the Admin UI facade and the External API will actually react on this, e.g. you cannot set an empty value for a required field using the Admin UI facade or the External API. Seem from that point of view, the comment "Display is optional" seems not very helpful and somewhat misleading as this depends on the actual value of the field "required"

Reply:
The intention was to show that the field could be commented out of the cfg file. Should the comment be removed altogether to avoid confusion, or just the "Display is" part?

Reply:
@karendolan Sorry... I did not read the description in detail enough... I thought that those fields would already be optional ;-) 

Nevertheless, thanks for removing the "Display is optional" comments as the metadata configuration has more effect than just "display" in Admin UI, so those comments were indeed somewhat misleading.